### PR TITLE
Unified buttons

### PR DESCRIPTION
**Pull request by oguzeroglu**
A unified look is implemented for buttons under modals for following classes:

btn
btn-primary
btn-secondary

including hover, active and disabled states.

Normal, hover and active states:
kap1

Disabled states:
Screenshot 2020-01-13 at 14 58 12

---

Closes https://github.com/camunda/camunda-modeler/issues/1627

### CODE REVIEW

File: client/src/app/primitives/modal/Modal.less [hunk 1]
```diff
@@ -1,4 +1,16 @@
 :local(.ModalOverlay) {
+  --blue-base: #4d90ff;
```
Comment (nikku) [inline client/src/app/primitives/modal/Modal.less]: Can we make these declarations global => Ensure that they are being used independent from where the `.btn-xxx` is being used?

**Comment by oguzeroglu** [comment]
Screenshot 2020-01-14 at 12 06 44

The button overflows are taken care of

**Comment by nikku** [comment]
Not working yet for start instance dialog (cf. button order):

image


Not working yet for keyboard shortcuts modal:

image

**Comment by oguzeroglu** [comment]
Keyboard Shortcuts is fixed 2 seconds before your comment :)

**Comment by nikku** [comment]
I've made the button styles global, too so that they can be reused: https://github.com/camunda/camunda-modeler/pull/1657/commits/eed35731e4e5653735faa3f701e37878dd09477c

**Pull request closed without merging**